poleno/utils/admin.py

```diff
@@ -37,3 +37,17 @@ def admin_obj_format(obj, format=u'{tag}', *args, **kwargs):
         except NoReverseMatch:
             pass
     return res
+
+class ReadOnlyAdminInlineMixin(object):
```

Comment:
@martinmacko47 pouzit dedenie `object` kedze to ma byt "Mixin", alebo priamo `admin.TabularInline`?

Reply:
Idealne to podedit z `InlineModelAdmin`. Aby to bolo pouzitelne aj pre `TabularInline` aj pre ine inline. Vid source `TabularInline` https://docs.djangoproject.com/en/1.8/_modules/django/contrib/admin/options/#TabularInline

Vo vseobecnosti pri mixinoch, ktore zavisia na nejakych metodach alebo properties inych class, je ich v tom mixine rovno podedit. Aby bolo jasne, v akom kontexte sa to pouziva. Z objectu by mali dedit iba uplne vsetobecne mixiny.

Reply:
:ok: